### DISCUSSION TITLE
Restrict content to A4 size

### DISCUSSION THREAD

**Discussion by andredewaard** [ueberdosis/tiptap#4488] [General Questions]
Is it somehow possible to restrict the content to only contain contents that could fit om a A4? 
i can set the width and height to that specific size but it would just overflow the contents you fill in more data.

**Comment by Cassielxd** [comment]
You can refer to this  https://github.com/Cassielxd/CassieEditor

**Reply by Screenplayink** [reply]
@Cassielxd I am looking to implement the emulation of A4 using your solution. I would like to virtually meet you. Can you please write to me at hello@screenplay.ink or ping me on discord  username : O.Brahma

**Reply by Cassielxd** [reply]
This is my email address 348040933@qq.com, you can contact me

**Comment by Screenplayink** [comment]


**Reply by Cassielxd** [reply]
How can I help you on your end

**Comment by Screenplayink** [comment]


**Reply by Cassielxd** [reply]
This is a quadratic extension implementation 
https://demo.umodoc.com/editor?lang=en-US

**Reply by Cassielxd** [reply]
Some known bugs have been fixed

**Reply by Cassielxd** [reply]
https://github.com/umodoc/editor

**Comment by Screenplayink** [comment]


**Reply by Cassielxd** [reply]
You add me to your organization. I'll take a look around

**Comment by Screenplayink** [comment]


**Reply by Cassielxd** [reply]
348040933@qq.com
